extensions/remote-udp/README.md

```diff
@@ -0,0 +1,67 @@
+# Remote UDP Extension
+
+The openEO API is a specification for interoperable cloud-based processing of large Earth observation datasets.
+
+This extension enables user to load user-defined processes that are hosted external to the openEO API (e.g., GitHub or cloud storage) through the process namespace into process graphs.
+
+- Version: **0.1.0**
+- Stability: **experimental**
+- Conformance class: `https://api.openeo.org/extensions/remote-udp/0.1.0`
+
+## Justification
+
+The openEO API defines the `namespace` property in a process node of a process graph as follows:
+>  The following options are predefined by the openEO API, but additional namespaces may be introduced by back-ends or in a future version of the API.
+> * `null` [...]
+> * `backend` [...]
+> * `user` [...]
+
+This makes it possible for this extension to add additional allowed values to the `namespace` property.
+
+## Specification
+
+This extension extends the `namespace` property of process graph nodes so that it accepts **absolute** URL with the protocols `https` (**recommended**) and `http` (discouraged). The URLs specified MUST either return
+
+1. a single process (compatible to the endpoint `GET /process_graphs/{process_graph_id}`).
+   In this case, the `id` property of the process graph node MUST be equal to the `id` of the process, 
+   otherwise a `ProcessNamespaceInvalid` error is thrown
+2. a list of processes (compatible to the endpoint `GET /process_graphs`)
+   In this case, the `id` property of the process graph node is used to identify the process from the list.
+   If not found a `ProcessNamespaceInvalid` error is thrown
```

Comment:
```suggestion
This extension extends the `namespace` property of process graph nodes so that it accepts **absolute** URL with the protocols `https` (**recommended**) and `http` (discouraged). The URLs specified MUST return one of the following two options:

1. A single process, compatible to the endpoint `GET /process_graphs/{process_graph_id}`.
   In this case, the `id` property of the process graph node MUST be equal to the `id` of the process, 
   otherwise a `ProcessNamespaceInvalid` error is thrown
2. A list of processes, compatible to the endpoint `GET /process_graphs`.
   In this case, the `id` property of the process graph node is used to identify the process from the list.
   If not found a `ProcessNamespaceInvalid` error is thrown
```